### DISCUSSION TITLE
Welcome to Fast-GitHub Discussions!

### DISCUSSION THREAD

**Discussion by fhefh2015** [fhefh2015/Fast-GitHub#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gametahenaiwo** [comment]
你好，商務合作，方便留個TG或者郵箱作為聯繫方式嗎？
